enterprise/server/auth/auth.go

```diff
@@ -226,9 +226,11 @@ type authenticator interface {
 type oidcAuthenticator struct {
 	oauth2Config *oauth2.Config
 	oidcConfig   *oidc.Config
-	provider     *oidc.Provider
+	Provider     *oidc.Provider
```

Comment:
Does Provider need to be a public member? (I'm probably missing where this is used...)